client/elife-theme/src/overrides/index.js

```diff
@@ -0,0 +1,3 @@
+import * as ui from './ui'
```

Comment:
Why did you call this overrides? I think it would be better to keep the name `elements` since everyone else (coko, epmc etc) are using elements?

Reply:
https://gitlab.coko.foundation/pubsweet/pubsweet/tree/master/packages/coko-theme/src
https://gitlab.coko.foundation/xpub/xpub-epmc/tree/dev/app/epmc-theme

Reply:
Because then the folder structure matches the object structure.